conf/config.ini

```diff
@@ -48,7 +48,7 @@ python_exec=
 ; Overridden by the KRESUS_WEBOOB_DIR environment variable, if it's set.
 ; Example:
 ; srcdir=/home/ben/code/weboob
-srcdir=__FINALPATH__/venv/bin/
+srcdir=__FINALPATH__/venv/lib/python2.7/site-packages/
```

Comment:
As discussed on framagit, I would suggest to set : 
- unset `srcdir` (leave it empty)
- set `python_exec=__FINALPATH__/venv/bin/python` 
Setting the python exe to the one in the virtual env will automatically set the PATH correctly (and could include other environnement variables which are not included with `srcdir`).